<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>950</width> <height>950</height> </rect> </property>
  <property name="windowTitle"> <string>تعديل بيانات القطعة</string> </property>
  <property name="layoutDirection"> <enum>Qt::RightToLeft</enum> </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget { font-family: 'Leelawadee UI', sans-serif; background-color: #F2F4F8; }
    #pageTitle { font-size: 32px; font-weight: 900; color: #000000; margin-bottom: 20px; }
    QScrollArea { border: none; background: transparent; }
    QWidget#scrollContents { background-color: transparent; }
    QGroupBox { background-color: #FFFFFF; border-radius: 16px; border: 1px solid #E0E0E0; margin-top: 30px; padding-top: 40px; padding-bottom: 25px; padding-left: 20px; padding-right: 20px; font-weight: 800; color: #000000; font-size: 18px; }
    QGroupBox::title { subcontrol-origin: margin; subcontrol-position: top left; padding: 5px 0px; background-color: transparent; color: #2962FF; }
    QLabel { color: #000000; font-size: 15px; font-weight: 800; margin-bottom: 5px; }
    QLineEdit, QComboBox, QDateEdit, QSpinBox, QDoubleSpinBox, QTextEdit { border: 2px solid #E0E0E0; border-radius: 10px; padding: 12px 15px; background: #FFFFFF; color: #000000; font-size: 15px; font-weight: 600; min-height: 25px; }
    QLineEdit:focus, QComboBox:focus, QTextEdit:focus, QSpinBox:focus { border: 2px solid #2962FF; background-color: #FFFFFF; }
    QListWidget { border: 2px solid #E0E0E0; border-radius: 10px; background: #FFFFFF; padding: 5px; }
    QPushButton { border-radius: 10px; padding: 12px 35px; font-weight: 900; border: none; font-size: 16px; }
    #btnSave { background-color: #F39C12; color: white; } #btnSave:hover { background-color: #E67E22; }
    #btnCancel { background-color: #7F8C8D; color: white; } #btnCancel:hover { background-color: #95A5A6; }
    #btnAddImages { background-color: #2962FF; color: white; padding: 10px 20px; font-size: 14px; }
    #btnRemoveImage { background-color: #FF1744; color: white; padding: 10px 20px; font-size: 14px; }
   </string>
  </property>
  
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin"> <number>0</number> </property>
   <item> <widget class="QScrollArea" name="scrollArea"> <property name="widgetResizable"> <bool>true</bool> </property> <widget class="QWidget" name="scrollContents">
      <layout class="QVBoxLayout" name="verticalLayout_3"> <property name="spacing"> <number>30</number> </property> <property name="margin"> <number>50</number> </property>
       <item> <widget class="QLabel" name="pageTitle"> <property name="text"> <string>تعديل بيانات القطعة</string> </property> </widget> </item>
       
       <item> <widget class="QGroupBox" name="groupBoxBasic"> <property name="title"> <string>1. التعريف الأساسي</string> </property> <layout class="QGridLayout" name="gridLayout"> <property name="verticalSpacing"> <number>25</number> </property> <property name="horizontalSpacing"> <number>25</number> </property> 
        <item row="0" column="0"> <widget class="QLabel" name="l1"> <property name="text"> <string>اسم القطعة *</string> </property> </widget> </item> <item row="0" column="1"> <widget class="QLineEdit" name="inputName"/> </item> <item row="0" column="2"> <widget class="QLabel" name="l_inv"> <property name="text"> <string>رقم الجرد *</string> </property> </widget> </item> <item row="0" column="3"> <widget class="QLineEdit" name="inputInventoryNo"/> </item> 
        <item row="1" column="0"> <widget class="QLabel" name="l2"> <property name="text"> <string>النوع</string> </property> </widget> </item> <item row="1" column="1"> <widget class="QComboBox" name="comboType"/> </item> <item row="1" column="2"> <widget class="QLabel" name="l3"> <property name="text"> <string>العدد</string> </property> </widget> </item> <item row="1" column="3"> <widget class="QSpinBox" name="spinQuantity"> <property name="minimum"> <number>1</number> </property> </widget> </item> 
        <item row="2" column="0"> <widget class="QLabel" name="l4"> <property name="text"> <string>المادة</string> </property> </widget> </item> <item row="2" column="1"> <widget class="QComboBox" name="comboMaterial"/> </item> <item row="2" column="2"> <widget class="QLabel" name="l_src"> <property name="text"> <string>المصدر</string> </property> </widget> </item> <item row="2" column="3"> <widget class="QLineEdit" name="inputSource"/> </item> </layout> </widget> </item>
       
       <item> <widget class="QGroupBox" name="groupBoxDims"> <property name="title"> <string>2. القياسات والوزن</string> </property> <layout class="QGridLayout" name="gridDims"> <property name="verticalSpacing"> <number>25</number> </property> <property name="horizontalSpacing"> <number>25</number> </property>
        <item row="0" column="0"> <widget class="QLabel" name="l_len"> <property name="text"> <string>الطول (سم)</string> </property> </widget> </item> <item row="0" column="1"> <widget class="QDoubleSpinBox" name="spinLength"/> </item> <item row="0" column="2"> <widget class="QLabel" name="l_wid"> <property name="text"> <string>العرض (سم)</string> </property> </widget> </item> <item row="0" column="3"> <widget class="QDoubleSpinBox" name="spinWidth"/> </item>
        <item row="1" column="0"> <widget class="QLabel" name="l_dia"> <property name="text"> <string>القطر (سم)</string> </property> </widget> </item> <item row="1" column="1"> <widget class="QDoubleSpinBox" name="spinDiameter"/> </item> <item row="1" column="2"> <widget class="QLabel" name="l_thk"> <property name="text"> <string>السمك (سم)</string> </property> </widget> </item> <item row="1" column="3"> <widget class="QDoubleSpinBox" name="spinThickness"/> </item>
        <item row="2" column="0"> <widget class="QLabel" name="l_wgt"> <property name="text"> <string>الوزن</string> </property> </widget> </item> <item row="2" column="1"> <layout class="QHBoxLayout"> <item> <widget class="QDoubleSpinBox" name="spinWeight"/> </item> <item> <widget class="QComboBox" name="comboWeightUnit"> <item> <property name="text"> <string>g</string> </property> </item> <item> <property name="text"> <string>kg</string> </property> </item> </widget> </item> </layout> </item> </layout> </widget> </item>

       <item> <widget class="QGroupBox" name="groupBoxLoc"> <property name="title"> <string>3. الموقع والتاريخ</string> </property> <layout class="QGridLayout" name="gridLoc"> <property name="verticalSpacing"> <number>25</number> </property> <property name="horizontalSpacing"> <number>25</number> </property>
        <item row="0" column="0"> <widget class="QLabel" name="l_store"> <property name="text"> <string>المستودع</string> </property> </widget> </item> <item row="0" column="1"> <widget class="QComboBox" name="comboStorage"/> </item> <item row="0" column="2"> <widget class="QLabel" name="l_row"> <property name="text"> <string>الصف</string> </property> </widget> </item> <item row="0" column="3"> <widget class="QLineEdit" name="inputStorageRow"/> </item> <item row="0" column="4"> <widget class="QLabel" name="l_col"> <property name="text"> <string>العمود</string> </property> </widget> </item> <item row="0" column="5"> <widget class="QLineEdit" name="inputStorageCol"/> </item> 
        <item row="1" column="0"> <widget class="QLabel" name="l5"> <property name="text"> <string>الفترة</string> </property> </widget> </item> <item row="1" column="1"> <widget class="QComboBox" name="comboPeriod"/> </item> <item row="1" column="2"> <widget class="QLabel" name="l6"> <property name="text"> <string>الحالة</string> </property> </widget> </item> <item row="1" column="3"> <widget class="QComboBox" name="comboCondition"/> </item> <item row="1" column="4"> <widget class="QLabel" name="l7"> <property name="text"> <string>تاريخ الحيازة</string> </property> </widget> </item> <item row="1" column="5"> <widget class="QDateEdit" name="dateRetrieval"/> </item> </layout> </widget> </item>
       
       <item> <widget class="QGroupBox" name="groupBoxDetails"> <property name="title"> <string>4. التوثيق والصور</string> </property> <layout class="QVBoxLayout" name="verticalLayout_4"> 
        <item> <widget class="QLabel" name="l9"> <property name="text"> <string>الوصف العلمي</string> </property> </widget> </item> <item> <widget class="QTextEdit" name="inputDescription"> <property name="maximumHeight"> <number>80</number> </property> </widget> </item> 
        <item> <widget class="QLabel" name="l11"> <property name="text"> <string>ملاحظات إضافية</string> </property> </widget> </item> <item> <widget class="QTextEdit" name="inputNotes"> <property name="maximumHeight"> <number>60</number> </property> </widget> </item> 
        <item> <layout class="QHBoxLayout" name="hImg"> <item> <widget class="QLabel" name="l10"> <property name="text"> <string>الصور المرفقة</string> </property> </widget> </item> <item> <spacer name="hss"> <property name="orientation"> <enum>Qt::Horizontal</enum> </property> </spacer> </item> <item> <widget class="QPushButton" name="btnAddImages"> <property name="text"> <string>+ رفع صور</string> </property> </widget> </item> <item> <widget class="QPushButton" name="btnRemoveImage"> <property name="text"> <string>- حذف</string> </property> </widget> </item> </layout> </item> 
        <item> <widget class="QListWidget" name="imagesList"> <property name="maximumHeight"> <number>100</number> </property> </widget> </item> </layout> </widget> </item>
       
       <item> <widget class="QGroupBox" name="groupBoxEditor"> <property name="title"> <string>5. بيانات التحرير</string> </property> <layout class="QHBoxLayout" name="editorLayout"> <property name="spacing"> <number>20</number> </property>
         <item> <widget class="QLabel" name="l_editor"> <property name="text"> <string>محرر البطاقة:</string> </property> </widget> </item> <item> <widget class="QLineEdit" name="inputEditor"/> </item> 
         <item> <widget class="QLabel" name="l_edit_date"> <property name="text"> <string>تاريخ التحرير:</string> </property> </widget> </item> <item> <widget class="QDateEdit" name="dateEditing"> <property name="calendarPopup"> <bool>true</bool> </property> <property name="displayFormat"> <string>yyyy-MM-dd</string> </property> </widget> </item> </layout> </widget> </item>

       <item> <layout class="QHBoxLayout" name="horizontalLayout_3"> <item> <spacer name="hSpacer"> <property name="orientation"> <enum>Qt::Horizontal</enum> </property> </spacer> </item> <item> <widget class="QPushButton" name="btnCancel"> <property name="text"> <string>إلغاء</string> </property> </widget> </item> <item> <widget class="QPushButton" name="btnSave"> <property name="text"> <string>حفظ التعديلات</string> </property> </widget> </item> </layout> </item>
      </layout> </widget> </widget> </item>
  </layout>
 </widget>
</ui>
